homework.md

```diff
@@ -44,37 +44,37 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used on the internet in websites for scripting, security and providing functionality.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Version Control, Sharing Code, Backing up projects.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make your own copy of another repo on your own Github profile.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You download the repo to your pc.
```

Comment:
what is i don't use pc? 

i think you should have generalized this by saying local machine / your local storage

Reply:
Well done @chandredewet  and @AdvocateM